libs/cart/src/drivers/magento/cart-shipping-address.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@angular/core';
+import { Apollo } from 'apollo-angular';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { DaffCartShippingAddressServiceInterface } from '../interfaces/cart-shipping-address-service.interface';
+import { DaffCart } from '../../models/cart';
+import { DaffCartAddress } from '../../models/cart-address';
+import { getShippingAddress, updateShippingAddress } from './queries';
+import { DaffMagentoShippingAddressInputTransformer } from './transforms/inputs/shipping-address.service';
+import { DaffMagentoCartTransformer } from './transforms/outputs/cart.service';
+import { DaffMagentoShippingAddressTransformer } from './transforms/outputs/shipping-address.service';
+import { MagentoGetShippingAddressResponse, MagentoUpdateShippingAddressResponse } from './models/responses';
+
+/**
+ * A service for making Magento GraphQL queries for carts.
```

Comment:
*for cart shipping addresses

Reply:
Changed in [`3f147a8`](https://github.com/graycoreio/daffodil/pull/670/commits/3f147a8f1be66ffebd9aab897807d1355b2c10ec).